AllergyIntolerance.000.ndjson

```diff
@@ -324,7 +324,7 @@
 {"resourceType":"AllergyIntolerance","id":"6a3efe1e-3a7b-9b4b-d2cd-c43647cc858c","meta":{"profile":["http://hl7.org/fhir/us/core/StructureDefinition/us-core-allergyintolerance"]},"clinicalStatus":{"coding":[{"system":"http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical","code":"active"}]},"verificationStatus":{"coding":[{"system":"http://terminology.hl7.org/CodeSystem/allergyintolerance-verification","code":"confirmed"}]},"type":"allergy","category":["food"],"criticality":"low","code":{"coding":[{"system":"http://snomed.info/sct","code":"735029006","display":"Shellfish (substance)"}],"text":"Shellfish (substance)"},"patient":{"reference":"Patient/51f823d8-d4d1-c96e-5f7d-1adc4bc428c4"},"recordedDate":"1939-09-02T09:04:01-04:00","reaction":[{"manifestation":[{"coding":[{"system":"http://snomed.info/sct","code":"247472004","display":"Wheal (finding)"}],"text":"Wheal (finding)"}],"severity":"mild"},{"manifestation":[{"coding":[{"system":"http://snomed.info/sct","code":"267036007","display":"Dyspnea (finding)"}],"text":"Dyspnea (finding)"}],"severity":"mild"},{"manifestation":[{"coding":[{"system":"http://snomed.info/sct","code":"271807003","display":"Eruption of skin (disorder)"}],"text":"Eruption of skin (disorder)"}],"severity":"moderate"},{"manifestation":[{"coding":[{"system":"http://snomed.info/sct","code":"39579001","display":"Anaphylaxis (disorder)"}],"text":"Anaphylaxis (disorder)"}],"severity":"severe"},{"manifestation":[{"coding":[{"system":"http://snomed.info/sct","code":"402387002","display":"Allergic angioedema (disorder)"}],"text":"Allergic angioedema (disorder)"}],"severity":"moderate"},{"manifestation":[{"coding":[{"system":"http://snomed.info/sct","code":"418290006","display":"Itching (finding)"}],"text":"Itching (finding)"}],"severity":"mild"},{"manifestation":[{"coding":[{"system":"http://snomed.info/sct","code":"49727002","display":"Cough (finding)"}],"text":"Cough (finding)"}],"severity":"mild"},{"manifestation":[{"coding":[{"system":"http://snomed.info/sct","code":"62315008","display":"Diarrhea (finding)"}],"text":"Diarrhea (finding)"}],"severity":"mild"}]}
```

Comment:
I don't know why these other resources decided to tweak themselves ever so slightly. :shrug: